ccloud/start.sh

```diff
@@ -61,7 +64,7 @@ confluent local start zookeeper
 sleep 2
 
 # Start local Kafka with Confluent Metrics Reporter configured for Confluent Cloud
-CONFLUENT_CURRENT=`confluent local current | tail -1`
+CONFLUENT_CURRENT=`confluent local current 2>&1 | tail -1`
```

Comment:
@rspurgeon is this a wrong commit from previous PR?

Reply:
It's necessary for "late model" `confluent` versions (as of Aug 2020).  It's snuck in here as I missed it for DEVX-1988.  I think it's harmless here but can be removed if you prefer.

Reply:
Ah, it was missed in previous PR.  👍 include here ;) LGTM